microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -111,11 +113,7 @@ Header Parameters: Authorization: Bearer $TOKEN; Content-Type: application/zip
 Body content: the package file to upload
 ```
 
-`$productID` is the ID of the Microsoft Edge Add-on that you want to update.  You can get the product ID in either of the following ways:
-
-*  Login to Microsoft Partner Center.  Navigate to **Microsoft Edge > Overview**, and then select the extension you want the product ID for.  The extension overview page opens.  The GUID in the URL is the product ID.
-
-*  Call the `/products` API to get a list of all products and their product IDs.  For more information about the `/products` API, navigate to [Get the list of products](addons-api-reference.md#get-the-list-of-products).
+`$productID` is the ID of the Microsoft Edge Add-on that you want to update. To get the product ID login to Microsoft Partner Center.  Navigate to **Microsoft Edge > Overview**, and then select the extension you want the product ID for.  The extension overview page opens.  The product ID key is available in the extension identity section.
```

Comment:
```suggestion
`$productID` is the product ID of the Microsoft Edge Add-on that you want to update. To get the product ID, login to Microsoft Partner Center. Go to **Microsoft Edge** > **Overview**, and then select the extension of the product ID you need.  The extension overview page opens.  The product ID key is available in the **Extension Identity** section.
```

Reply:
I am unable to reproduce the User Interface steps. I have access to Partner Center, but have not used it. If someone can verify the UI steps or take a screen shot of the path or destination page that would be great.